libs/labelbox/pyproject.toml

```diff
@@ -74,6 +74,10 @@ dev-dependencies = [
 [tool.ruff]
 line-length = 80
 
+[tool.ruff.lint]
+ignore = ["F", "E722"]
```

Comment:
I have PyFlack and a bareexcept rule ignored here. Later we should look at not ignoring these rules and possible include more then just the defaults